test/Utils.purs

```diff
@@ -1,27 +1,26 @@
 module Test.Utils (unsafeCall, assertTrue, assertTrue_, errMaybe) where
 
-import Control.Alternative (pure)
+import Control.Alternative (class Applicative, pure)
 import Data.Function (($))
 import Data.Maybe (Maybe(Just, Nothing))
 import Data.Unit (Unit, unit)
 import Effect.Aff (error)
 import Effect.Class (class MonadEffect, liftEffect)
 import Effect.Exception (throwException, throw)
-import TestM (TestPlanM)
 import Type.Prelude (Proxy)
 
 foreign import unsafeCall :: forall a b. Proxy b -> String -> a -> b
 
 -- | Make boolean a test
-assertTrue :: String -> Boolean -> TestPlanM Unit
+assertTrue :: forall (m :: Type -> Type). Applicative m => MonadEffect m => String -> Boolean -> m Unit
```

Comment:
Why a kind annotation here? I'm so used to reading types like this without it I doubt it provides value. Plus it's obvious from `Applicative` constraint.

Reply:
I was asked to add kind signatures by @vvtran , I think it follows haskells style guide, I don't like it either, unless necessary.